editor/components/default-block-appender/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import classnames from 'classnames';
+import 'element-closest';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { getDefaultBlockName, createBlock } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import BlockDropZone from '../block-drop-zone';
+import { insertBlock } from '../../actions';
+import { getBlockCount } from '../../selectors';
+
+class DefaultBlockAppender extends Component {
+	constructor( props ) {
+		super( props );
+		this.appendDefaultBlock = this.appendDefaultBlock.bind( this );
+	}
+
+	appendDefaultBlock() {
+		const newBlock = createBlock( getDefaultBlockName() );
+		this.props.onInsertBlock( newBlock );
+	}
+
+	render() {
+		const { count } = this.props;
+		const className = classnames( 'editor-default-block-appender', {
+			'is-visible-placeholder': count === 0,
+		} );
+
+		return (
+			<div className={ className }>
+				<BlockDropZone />
+				<button onClick={ this.appendDefaultBlock }>
+					{ count === 0 && __( 'Write your story' ) }
+				</button>
+			</div>
+		);
+	}
+}
+
+export default connect(
```

Comment:
it would be nice to have 3 unit tests:

- count > 0 (snapshot test)
- count = 1 check if there is class `is-visible-placeholder` and `Write your story` text
- spy on `insertBlock`

I can add them, just ping me :)